students/TacSPx/lesson04/mailroompart2.py

```diff
@@ -69,38 +64,35 @@ def thank_you():
     '''
     while True:
         donor_name = input("Type 'List' to see all names in Database or\nType First and Last Name: ").title()
-        if (donor_name.lower() == 'list'):
+        if donor_name.lower() == 'list':
             print("\nDONOR REGISTRY:")
             donor_sort = (sorted(donor_info.keys()))  # used .keys() to sort on key
             for key in donor_sort:
                 print(key)
             input("Press Enter to Continue")
             print()
-            continue
 
-        if (donor_name.lower() != 'list'):
+        else:
             donor_amount = int(input("Enter Amount of Donation: $"))
             # if name is in dict append dollar amount to donors bucket
-            for i in donor_info:
-                if i == donor_name:
-                    donor_info[donor_name].append(donor_amount)
-                    print(f"'{donor_name}' is in registry added new donation amount!")
-                    input("\nPress enter to continue to 'Thank You' letter:")
-                    # print(donor_info)  # - for testing
-                    break
-                # if name not in list add new name to donor info list
+            if donor_name in donor_info:
```

Comment:
since you're checking for a key in dictionary the correct way to check is via get()

so `donor_info.get(donor_name)`